### ISSUE TITLE
Team update - Aug 31, 2022

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#156]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

* [Rendered notebook user testing - Round 1](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/issues/146)

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌**

- @tonyfast thank you for continuing to know and share all the Jupyter things. Things I make wouldn't be nearly as useful without your (metaphoric) hikes.

**Updates from last week :heavy_check_mark:**

- #146. User tests for this round conclude this week! Next week issues will start and I can share some of our early findings.
- #133. I made a little bit of progress on this issue on a few fronts. 
     - First and most important is [jupyterlab-accessible-themes #34's comment about how the main areas of JupyterLab could react in high-zoom/low space situations.](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/34#issuecomment-1228911639). I think we need to make a choice about our approach as a team to solve the problems that happen within each space.
     - I noted [an additional issue with 400% zoom in the Running tab](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/3#issuecomment-1232025460) I found when trying to test and propose solutions for…
     - the [written and tested version of last week's open tab navigation discussion](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/9#issuecomment-1232080309). I was wrong! There is some amount of things that exist for open tab navigation, but I do still think it could be optimized.

**Challenges I faced and things I'd like assistance with 🙏**
- I don't think I have anything at the moment.

**My availability for next week**
- I'll be out of office September 1–September 5.

**Important items for discussion 💬**
- This can be an asynchronous discussion, but I think the 400% zoom reflow is somewhat blocking everyone as long as we haven't decided how all the JupyterLab components fit together. So I'd suggest giving [comment on quansight-labs/jupyterlab-accessible-themes](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/34#issuecomment-1228911639) a review if you can.

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- Thanks to @isabela-pf and @steff456 for being so proactive and hands-on helping Kulsoom scope an issue to work. Really makes me feel supported by a team ☺️

**Updates from last week :heavy_check_mark:**
- https://github.com/Quansight-Labs/jupyter-a11y-testing/issues/13. Put up draft PR, still more to do.
- Worked with Kulsoom to help identify and scope an accessibility issue.
- https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/34#issuecomment-1228911639. Reviewed Isabela's zoom/reflow sketches.

**My availability for next week**
- Available all next week!

**Important items for discussion 💬**
- Ditto what Isabela said: let's discuss approaches to zoom/reflow with an eye towards scoping work for Kulsoom?
- From my 1-on-1 earlier this with Tania, it seems that we're going to want to really make the Quansight-Labs/jupyter-a11y-testing capable of providing detailed reports on the state of accessibility in JupyterLab. My question is whether I should split my time between that and the theme extension work, or if Stephannie will own the theme extension and I will own the testing app/service?

**Comment by steff456** [comment]
**Thanks I'd like to give 🙌**
- @isabela-pf for always helping me!

**Updates from last week :heavy_check_mark:**
- I worked towards fixing the logo issue
  - Created a PR adding the svg asset to ipykernel: https://github.com/ipython/ipykernel/pull/984
  - Created a PR adding the R asset to irkernel
  - Scheduled a meeting with Mike for a code overview in jupyterlab to understand better the manager architecture.
- Reviewed Gabriel's PR: https://github.com/Quansight-Labs/jupyter-a11y-testing/issues/13
- Tried to get Kulsoom on track with a scoped work

**Challenges I faced and things I'd like assistance with 🙏**
- I think we need to assign a first good issue to Kulsoom so she isn't blocked until Isabela comes back from PTO.

**My availability for next week**
- I'll be here

**Important items for discussion 💬**
- Kulsoom's first issue

**Comment by kulsoomzahra** [comment]
**Thanks I'd like to give 🙌**
- @isabela-pf @gabalafou @steff456 for their immense support & making my life easier! Thanks a ton :hugs: 

**Updates from last week :heavy_check_mark:**
- Brainstorming with Stephannie and Gabriel towards figuring out what part of the [Zoom](https://github.com/jupyterlab/jupyterlab/issues/10004) issue I could manage
- Tried to gather more insights from Isabela around the ["jump to tab"](https://quansight.slack.com/archives/C02HG0AHA9H/p1661409730008789?thread_ts=1661190488.556999&cid=C02HG0AHA9H) functionality
- Spent some time reading [Reflow criterion (1.4.10) of WCAG 2.1](https://drive.google.com/file/d/1VYgRrtCckUMAj_3A21FY3DPei0YYtxd5/view?usp=sharing_eil&ts=5e177beb)

**Challenges I faced and things I'd like assistance with 🙏**
- I'm hoping to get some more input to actually start implementing the "jump" button

**My availability for next week**
- I'll be here

**Comment by trallard** [comment]
@kulsoomzahra: this feels like a slow week - doing loads of prep work for Zoom fixes
- She still needs some more input from @isabela-pf  to move forward with the [Jump to tab implementation](https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/9#issuecomment-1232080309)
- to sync with @steff456  to find out how to fix vertical alignment at 400% Zoom https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/9
- open an issue with the task + steps to tackle the issue

@steff456: 
- working on the icon issue: she already has some PR's opened
- meeting with Mike to go over the manager architecture
